addons/pos_restaurant/static/src/js/Chrome.js

```diff
@@ -10,7 +10,7 @@ patch(Chrome.prototype, "pos_restaurant.Chrome", {
      * `FloorScreen` is the start screen if there are floors.
      */
     get startScreen() {
-        if (this.env.pos.config.iface_floorplan) {
+        if (this.env.pos.config.iface_floorplan || this.env.pos.config.is_table_management) {
```

Comment:
Don't forget to remove this one because is_table_management will replae iface_floorplan